sdks/universal-router-sdk/test/uniswapTrades.test.ts

```diff
@@ -1604,7 +1617,7 @@ describe('Uniswap', () => {
           tickUpper: 300000,
         }),
         outputPosition: new V4Position({
-          pool: WETH_USDC_V4,
+          pool: ETH_USDC_V4,
```

Comment:
all of the successful migration tests are now WETH->ETH tests, theres no longer any tests of ERC20->ERC20 regular migration case